### ISSUE TITLE
Physical interpretation of buried depth

### ISSUE THREAD

**Issue by wouterpeere** [MassimoCimmino/pygfunction#268]
I was looking into the effect of the buried depth on the gfunctions and I was struggling to come up with a physical meaning for this.

If I understood it correctly, the effect of the buried depth on the final gfunction is more pronounced if the borehole depth is rather small (so for larger D/H ratios) (Cimmino et al., 2013). From a physical point of view, I interpret this as the buried depth forming an 'insulation layer' between the boreholes and the surface temperature. The bigger this layer w.r.t. borehole depth, the more 'insulated' the field becomes and the more pronounced the temperature effects (and hence) the gfunctions will be. A smaller D/H ratio can then be seen as a sort of 'dampening' effect on the temperature response, since there is relatively more thermal interaction with the surface temperature.

I was wondering: this does assume that the surface temperature is at the same reference temperature as the ground, right?
But if you have a surface temperature of 10°C, a gradient of 2K/100m and a borehole depth of 200m, the average ground temperature will be around ~11°C, whereas the surface temperature is at a reference of 10°C. Does this not cause any losses in accuracy?

Best regards
Wouter Peere

**Comment by j-c-cook** [comment]
The unit-step thermal response g-function is calculated with a pure conduction based model. The homogeneous medium is initially considered to be at a single initial temperature. The thermal propagation is based on the finite line source, which constantly rejects heat. The g-function was developed to be decoupled from the specific phenomena due to the computational effort. A g-function could be pre-computed and stored in a database to be later used in a simulation. 

The [simulation](https://github.com/j-c-cook/ghedt/blob/1e04839a2329cbd44d329ecf0bf72da00d346943/ghedt/ground_heat_exchangers.py#L162-L178) in `ghedt` considers a single undisturbed ground temperature. There's been limited work on the [validation of design methods](https://www.researchgate.net/publication/276834112_Validation_of_vertical_ground_heat_exchanger_design_methodologies). Likely due to the high cost associated with drilling. (I was recently quoted at $60,000 for a GSHP with 3 boreholes each at 200 ft. Based on that contractor's quote, I'd estimate it would cost at least $30k out of pocket if I did as much of the work myself as I could handle.)  [Thermal response tests](https://en.wikipedia.org/wiki/Thermal_response_test) are used in practice if the system to be installed is _large_, which can help to quantify the effective borehole thermal resistance. Perhaps the temperature variation along the z-axis is somewhat captured in these tests. 

TLDR; There is unknown and uncertainties in the design process. Modeling every physical phenomena is computationally demanding, so shortcuts are taken to enable design. The _accuracy_ impact of those shortcuts are hard to quantify because the amount of experimental data is limited.

**Comment by wouterpeere** [comment]
Hi @j-c-cook ,

Thanks for your response! I was aware of a couple of the things you mentioned and I agree that there should ideally be more experimental data to quantify the different shortcuts in sizing. Thermal response tests can give you indeed an idea of the undisturbed ground temperature, which has a very big impact in the final design [(Radioti et al., 2017)](https://www.sciencedirect.com/science/article/abs/pii/S0306261917305639).

My question however was more related towards the theoretical aspect of taken the buried depth into account. What is, from a mathematical point of view, the importance of the assumption that the ground and surface temperature are equal?

Best,
@wouterpeere

**Comment by MassimoCimmino** [comment]
For the `UHTR` and `UBWT` boundary conditions, the gradient can simply be superimposed and the ground temperature in simulations is taken at the mid-length of the borehole. The surface temperature would still be constant but equal to the corrected ground temperature considering the temperature gradient. There should not be any loss in accuracy in these cases, as the linearity of the heat transfer problem allows to superimpose the gradient and the solutions while still respecting the boundary conditions. When using the `UBWT` boundary condition, this would assume that the borehole wall temperature decreases linearly with depth and is no longer linear. If I recall correctly, this was considered by Eskilson (1987) in their thesis.

This is not the case for the `MIFT` boundary condition, since the temperature gradient would affect the distribution of heat transfer rates along boreholes. How much this impacts on the g-function is, as far as I am aware, an open question.

For varying surface temperatures, the effect can be superimposed without much loss in accuracy if the boreholes are deep enough. I had done some analysis on that in a previous conference publication, here : [IGSHPA 2017 research proceedings](https://igshpa.org/wp-content/uploads/2017-IGSHPA-Research-Track-Proceedings-1.pdf).

**Comment by j-c-cook** [comment]
@wouterpeere Have you managed to superimpose a ground temperature gradient and compare the resulting error on the g-functions for either a `UHTR` or `UBWT`?

Is there any remaining items to address with this issue? Is there a desire to further investigate whether the ground temperature gradient could be superimposed for the `MIFT` boundary condition?
